tips/models.py

```diff
@@ -8,20 +8,11 @@ class Tip(models.Model):
     text = models.TextField(max_length=200)
     date = models.DateTimeField(default=timezone.now)
 
-    category = models.CharField(
-         max_length=32,
-         choices=Expenses.Category.choices,
-         default=Expenses.Category.OTHER
-    )
+    category = models.CharField(max_length=32, choices=Expenses.Category.choices, default=Expenses.Category.OTHER)
```

Comment:
Why did you change this model? 
He is not tied to the house

Reply:
Lior just rearanged this line, The data stayed the same :)